platform/cli/templates/extension/dependencies.json

```diff
@@ -57,6 +57,6 @@
     "eslint-loader": "^2.0.0",
     "webpack": "^5.50.0",
     "webpack-merge": "^5.7.3",
-    "webpack-cli": "^4.7.2"
+    "webpack-cli": "^5.0.2"
```

Comment:
i think we are using another version elsewhere

Reply:
Updating that overall, so leaving this one.